FrameworkSystem/private/standardLogging/Handler/MessageQueueHandler.py

```diff
@@ -37,6 +37,9 @@ def __init__(self, queue):
     result = createProducer(queue)
     if result['OK']:
       self.producer = result['Value']
+    else:
+      # print because logging not available
+      print("ERROR initializing MessageQueueHandler: %s" % result)
```

Comment:
```suggestion
      print("ERROR initializing MessageQueueHandler:", result)
```

Reply:
What's wrong with the `%s` ?

Reply:
I think it's better style to use multiple arguments to `print` as `%` formatting is longer, harder to lint and more likely to result in an exception due to accidentally passing the wrong number of parameters.

That said obviously in this case it makes no difference as I'm 99% sure it's correct as-is so I'm happy either way.

Reply:
yeah, the multitude of way to format string does not really help... between the C-style, the multiple number of arguments, and the `.format`....

Reply:
You missed the one true way! (`f"Hello {name}"`) 😆

Reply:
True ! :D